.ci/ansible/roles/centos/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: Enable centos-release-scl repo
+  yum: name=centos-release-scl state=latest
+
+- name: Setup libvirt
+  yum: name={{item}} state=latest
```

Comment:
Does this playbook going to run everytime to provision same centos box? if yes then better to use 'present' to make sure different run doesn't change the version. if box itself created for every run then it's fine.